data/discord/messages/homeassistant.yaml

```diff
@@ -1375,3 +1375,19 @@ chatgpt:
     convincing looking answers... that are either totally garbage or, if you're lucky,
     have subtle flaws. [Don't use them](https://www.home-assistant.io/blog/2023/01/23/help-others-leave-ai-at-the-dor/)
     to "help" others, and don't use them yourself.
+
+safe_mode:
+  description: "Enable safe mode"
+  content: >-
+    You can restart your Home Assistant instance in safe mode.
+    This will disable the loading of anything custom, like custom integrations, custom cards, custom frontend modules, and custom themes.
+
+    This is useful to enable you to rule out potential issues caused by running custom _things_.
+
+    You can enable safe mode by going to [Settings -> System](<https://my.home-assistant.io/redirect/system_dashboard/>)
+    in your Home Assistant UI, then clicking the power button in the top right corner,
```

Comment:
```suggestion
    in your Home Assistant UI, then selecting the power button in the top right corner,
```

Reply:
```suggestion
    in your Home Assistant UI, then hitting the power button in the top right corner,
```

Reply:
```suggestion
    in your Home Assistant UI, then touching the power button in the top right corner,
```

Reply:
```suggestion
    in your Home Assistant UI, then using the power button in the top right corner,
```

Reply:
🤷‍♂️

Reply:
I mean, I am now being taught by Frenck and Coffeecat that we should use device unrelated wording in docs so why not here as well 🤷🏻